README.md

```diff
@@ -11,13 +11,14 @@ files as well as several neutral formats (csv, hdf5, vtk,
 etc.). Various operators are available allowing the manipulation and
 the transformation of this data.
 
-The Python `ansys.dpf.post` package provides an simplified Python
+The Python `ansys-dpf-post` package provides an simplified Python
 interface to DPF, thus enabling rapid post-processing, without ever
 leaving a Python environment. 
 
-This module leverages the DPF-Core project's ``ansys.dpf.core``
-package and can be found by visiting [DPF-Core GitHub](https://github.com/pyansys/DPF-Core).  Use ``ansys.dpf.core``
-for building more advanced and customized workflows using Ansys's DPF.
+This module leverages the DPF-Core project's ``ansys-dpf-core`` package and can
+be found by visiting [PyDPF-Core
+GitHub](https://github.com/pyansys/pydpf-core).  Use ``ansys-dpf-core`` for
+building more advanced and customized workflows using Ansys's DPF.
```

Comment:
@akaszynski Grammatically, Ansys's DPF is correct. However, all corporate content these days is using Ansys' as the possessive form. (This is wrong because you naturally pronounce as "Ansys-es".) Should we leave as is or reflect what is put out by our corporate communications team?

Reply:
> Jenna told me that it had to be updated in TFS and pushed to the repo?

I think we're moving all dev for DPF into GitHub (@PProfizi, please verify).

Leave as-is and then commit to a follow-up PR.